### PR TITLE
Document installation from conda-forge

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

**Comment by asottile** [comment]
@nicoddemus does this look right?

File: index.mako [hunk 1]
```diff
@@ -79,9 +79,17 @@ pre-commit
...
-```
+```bash
 brew install pre-commit
 ```
+
+Using [conda](https://conda.io) (via [conda-forge](https://conda-forge.org)):
+
+```bash
+conda config --add channels conda-forge
+conda install pre_commit
```
Comment (nicoddemus) [inline index.mako]: I think it is better to advertise:

```
conda install -c conda-forge pre_commit
```

This is what's suggested [here](https://anaconda.org/conda-forge/pre_commit), also the former would change the global configuration which is not very nice.

Comment (asottile) [reply index.mako]: nice! pretty clear that

**Comment by asottile** [comment]
@nicoddemus thanks for the review!

**Comment by nicoddemus** [comment]
sure, anytime!

**Comment by asottile** [comment]
@nicoddemus btw the directions I grabbed were from https://github.com/conda-forge/pre_commit-feedstock#installing-pre_commit

**Comment by nicoddemus** [comment]
Hmm I see. That's probably because conda-forge wants to advertise for users to switch to its channel for good. 👍

**Pull request merged**